test/parallel/test-buffer-readuintle.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+// testing basic functionality of readUIntLE()
+
+const buf = Buffer.from([42, 84, 168, 127]);
+const result = buf.readUIntLE(2);
+
+assert.strictEqual(result, 168);
+
+assert.throws(
+      () => {
```

Comment:
@jasnell Can you be more specific? To my mind, this would be OK if the indentation on line 13 was two spaces rather than six spaces. Is that your opinion too?